### PR TITLE
[#10564] Feedback session question options: unable to select text by dragging cursor

### LINKED ISSUE #10564: Feedback session question options: unable to select text by dragging cursor
- **Environment**: master, Chrome on macOS

**Steps to reproduce**

image
Unable to select the text on the options by pressing down mouse and dragging the cursor. Instead it would start reordering the options. Affects distribute point (among options) question and rank options question.

**Suggested change**

Change the draggable area to the arrow icon only?

### PR DESCRIPTION
**Pull request by AdithyaNarayan**
Fixes #10564 

**Outline of Solution**
- Extracted the drag button from the `rank-options-field` component and placed in in the `rank-options-question-edit-details-form`
- Extracted the drag button from the `constsum-options-field` component and placed in in the `constsum-options-question-edit-details-form`

**Expected Behaviour of the drag options**

working gif

### CODE REVIEW

**Review by niqiukun — CHANGES_REQUESTED** [review]
Try to avoid making unnecessary UI changes here. For example, the input bar used to be longer and there is no space between the handle and the input bar before this PR is applied.

Original:
image

After change:
image

**Comment by AdithyaNarayan** [comment]
Oh sorry! I think I have fixed it now!

File: src/web/app/components/question-types/question-edit-details-form/constsum-options-question-edit-details-form.component.html [hunk 1]
```diff
@@ -3,15 +3,18 @@
   <div class="col-sm-6">
     <div class="row">
       <div id="options-section" class="col-sm-9" cdkDropList (cdkDropListDropped)="onConstsumOptionDropped($event)" [cdkDropListDisabled]="!isEditable">
-        <div *ngFor="let option of model.constSumOptions; let i = index;
+        <div class="row margin-bottom-20px padding-left-12px d-flex" *ngFor="let option of model.constSumOptions; let i = index;
```
Comment (hcwong) [inline src/web/app/components/question-types/question-edit-details-form/constsum-options-question-edit-details-form.component.html:6]: is `d-flex` necessary? I don't see any UI changes when I remove that CSS class.

**Review by hcwong — APPROVED** [review]
Looks fine to me, except for that one comment

**Review by madanalogy — APPROVED** [review]
LGTM

**Comment by madanalogy** [comment]
@niqiukun need your approval on this since you requested changes

**Review by niqiukun — APPROVED** [review]
LGTM!

**Pull request merged**